src/App.js

```diff
@@ -1,5 +1,26 @@
+import { Console } from "@woowacourse/mission-utils";
+
+import { parseInput } from "./inputParser.js";
+import { calculateSum } from "./calculator.js";
+
 class App {
-  async run() {}
+  async run() {
+    try {
+      const input = await this.getInput();
+      const numbers = parseInput(input);
+      const result = calculateSum(numbers);
+      this.printResult(result);
+    } catch (error) {
+      throw new Error("[ERROR] " + error.message);
+    }
+  }
+
+  async getInput() {
+    return await Console.readLineAsync("덧셈할 문자열을 입력해 주세요.\n");
+  }
+  printResult(result) {
+    return Console.print("결과 : " + result);
+  }
```

Comment:
```suggestion
  printResult(result) {
    Console.print("결과 : " + result);
  }
```
여기 메서드의 목적은 콘솔에 결과를 출력하는 것 같은데 지금은 `콘솔에 결과도 출력`하고, `결과를 출력하는 콘솔 로직을 리턴`하는 것 같아서 `return`을 빼는게 조금 더 목적에 맞지 않나? 라는 생각이 드는데 지현님은 어떻게 생각하시나요??

지금 방식은 이렇게도 사용가능해서 목적에는 조금 달라질 수도 있을 것 같아요!
```js
const consoleFn = printResult(1); // 콘솔에 이미 결과 : 1이 출력됨
consoleFn(); // 여기서 리턴 받은 콘솔 함수를 실행하면 한번 더 결과: 1을 출력할 수 있음
```

Reply:
아차차.. 중요한 부분인것을 간과하고 말았습니다. 😇 저도 말씀해주신 방향에 공감합니다!! 
식별자 명에 알맞지 않는 로직인것과 함께, 제가 아닌 다른 분이 메서드 및 반환값을 어떻게 사용할 지 모른다는 점에서 문제를 야기할 수 있는 코드네요.
리뷰로 남겨주셔서 너무 감사합니다!! 놓치고 지나갈뻔 했습니다.🙇‍♂️